DirectEmployers/MyJobs
# Pd 3531 method for selecting dimensions

## What's this PR do?
Prototype for traffic filters in Analytics 

## Screenshots (if appropriate)
![screen shot 2017-08-08 at 2 55 56 pm](https://user-images.githubusercontent.com/25237191/29089278-bee1cd92-7c49-11e7-9872-65bd7a2d7334.png)

## How should this be manually tested?
Pull down branch > open Analytics > First Viewed On > Click caret to show Traffic Sources filters > Select filters > Clear fields by clicking 'x'

Filters should show active or disabled depending on which filter is selected.

## Any background context you want to provide?
This is a prototype for the demo. Toggling will be implemented. Functionality for Technology Combos will be implemented.
